### PR TITLE
Rename browser/execution context to browsing context

### PR DESCRIPTION
**Pull request by tQsW**
PR of #23

### CODE REVIEW

File: draft-ietf-rtcweb-mdns-ice-candidates.md [hunk 1]
```diff
@@ -104,8 +104,8 @@ are not exposed to the web application.
 Principle {#principle}
 ============
 
-This section uses the concept of ICE agent as define in {{RFC8445}}.
-In the remainder of the document, it is assumed that each browser execution context has its own ICE agent.
+This section uses the concept of ICE agent as defined in {{RFC8445}}. In the
+remainder of the document, it is assumed that each browsing context has its own ICE agent.
```
Comment (tQsW) [inline draft-ietf-rtcweb-mdns-ice-candidates.md]:
```suggestion
remainder of the document, it is assumed that each [browsing context](https://html.spec.whatwg.org/multipage/browsers.html#browsing-context) has its own ICE agent.
```

Comment (juberti) [reply draft-ietf-rtcweb-mdns-ice-candidates.md]: Alas, you'll need to ref this IETF-style, similar to how WebRTCSpec is linked.

**Comment by juberti** [comment]
Anyone else want to take a look?

File: draft-ietf-rtcweb-mdns-ice-candidates.md [hunk 2]
```diff
@@ -228,14 +232,14 @@ beginning of ICE gathering;
 - the addition of remote ICE host candidates with mDNS names generates mDNS
 queries for names of each candidate;
 
-- the removal of names could happen when the execution context of the ICE agent
+- the removal of names could happen when the browsing context of the ICE agent
 is destroyed in an implementation, and goodbye responses should be sent to
```
Comment (jeroendeb) [inline draft-ietf-rtcweb-mdns-ice-candidates.md]: Shouldn't should be SHOULD? E.g. make it a recommendation to invalidate mDNS records?

Comment (jeroendeb) [reply draft-ietf-rtcweb-mdns-ice-candidates.md]: That's probably to strong considering we allow mDNS names to be pre-generated/allocation.

**Pull request merged**